Jamfile

```diff
@@ -491,6 +491,7 @@ variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
 	  <export-extra>on <debug-iterators>on <threading>multi
+	  <inlining>off
```

Comment:
The real reason is that `boost-build` in brew is very very old, it specifies the flag `-finline-functions` in release by default, but newer versions of clang can't accept this flag. Now, I found that you already specify `<optimization>off` because of travis issues, so I thought that one less optimization is of no harm here.

You tell me if this is acceptable.